shared/search/user-pane/user.render.desktop.js

```diff
@@ -27,18 +28,19 @@ export default class Render extends Component<void, Props, void> {
           <UserProofs
             style={{marginTop: globalMargins.small, marginLeft: globalMargins.medium, marginRight: globalMargins.medium}}
             username={this.props.username}
+            loading={this.props.loading}
             proofs={this.props.proofs}
             currentlyFollowing={this.props.currentlyFollowing}
           />
         </Box>
-        <UserActions
-          style={styleActionBox}
-          trackerState={this.props.trackerState}
-          currentlyFollowing={this.props.currentlyFollowing}
-          onFollow={this.props.onFollow}
-          onUnfollow={this.props.onUnfollow}
-          onAcceptProofs={this.props.onAcceptProofs}
-        />
+        {!this.props.loading &&
```

Comment:
we should coordinate as i'm also injecting an idea of loading but its just for the serverActive piece to control the action bar